tenable/io/scans.py

```diff
@@ -849,6 +849,13 @@ def results(self, scan_id, history_id=None, history_uuid=None):
             params['history_uuid'] = self._check(
                 'history_uuid', history_uuid, 'scanner-uuid')
 
+        if 'web-app2' in kw.values():
```

Comment:
Please document the new kew word argument - `web-app2` for the `readthedocs` doc generation.

You could take the following as your example:

```
scan_type (str, optional):
                This parameter is required only when using the API with
                Web Application Scanning. Available option is 'web-app'.
```